content-services/latest/support/index.md

```diff
@@ -81,7 +81,7 @@ Choose a combination of products to build your own Supported Stack. If anything
 | Alfresco Content Connector for AWS S3 4.1 | |
 | Alfresco Content Connector for Azure 2.1 | |
 | Alfresco Content Connector for AWS Glacier 2.2 | |
-| Alfresco Content Connector for Salesforce 2.3.1 | |
+| Alfresco Content Connector for Salesforce 2.3.2 | |
```

Comment:
Based on the Release Notes, it looks like all Salesforce Connector 2.3.x releases are supported on 6.2.2 onwards.
```suggestion
| Alfresco Content Connector for Salesforce 2.3 | |
```